package Exception;
import java.util.Scanner;
import java.io.File;
import java.io.IOException;

/**
 *
 * @author SyNguyen
 */
public class KiemTraTamGiac {
    public static void main(String[] args) {
//        File file1 = new 
//        Scanner sc = new Scanner(System.in);
//        float a,b,c;
//        System.out.println("Nhap a: ");
//        a = sc.nextFloat();
//        System.out.println("Nhap b: ");
//        b = sc.nextFloat();
//        System.out.println("Nhap c: ");
//        c = sc.nextFloat();
//        if ((a+b>c) && (b+c>a) && (a+c>b)) {
//            System.out.println("True");
//        } else {
//            System.out.println("False");
//        }
        try {
            int[] a = {1,2,3,4,5};
            int n = 10;
            for (int i =0; i<n;i++) {
                System.out.print(val);
            }
            System.out.println("");
        } catch (SecurityException e) {
            System.out.println("Loi chi so mang ");
        } 
        System.out.println("");
        System.out.println("Chay tiep bthg");
    }
}
